Fix IndexError when using CondorProvider with strategy enabled

```
2019-09-20 15:32:16.535 parsl.dataflow.flow_control:142 [ERROR]  Flow control callback threw an exception - logging and proceeding anyway
Traceback (most recent call last):
  File "/afs/crc.nd.edu/user/a/awoodard/.local/lib/python3.6/site-packages/parsl/dataflow/flow_control.py", line 140, in make_callback
    self.callback(tasks=self._event_buffer, kind=kind)
  File "/afs/crc.nd.edu/user/a/awoodard/.local/lib/python3.6/site-packages/parsl/dataflow/strategy.py", line 177, in _strategy_simple
    status = executor.status()
  File "/afs/crc.nd.edu/user/a/awoodard/.local/lib/python3.6/site-packages/parsl/executors/high_throughput/executor.py", line 620, in status
    status = self.provider.status(list(self.blocks.values()))
  File "/afs/crc.nd.edu/user/a/awoodard/.local/lib/python3.6/site-packages/parsl/providers/condor/condor.py", line 160, in status
    self._status()
  File "/afs/crc.nd.edu/user/a/awoodard/.local/lib/python3.6/site-packages/parsl/providers/condor/condor.py", line 142, in _status
    job_id = parts[0]
IndexError: list index out of range
```

--- a/parsl/providers/condor/condor.py
+++ b/parsl/providers/condor/condor.py
@@ -143,8 +143,8 @@ class CondorProvider(RepresentationMixin, ClusterProvider):
         34524643.0 1
         """
 
-        for line in stdout.strip().split('\n'):
-            parts = line.split()
+        for line in stdout.splitlines():
+            parts = line.strip().split()
             job_id = parts[0]
             status = translate_table.get(parts[1], 'UNKNOWN')
             self.resources[job_id]['status'] = status

